auth-api/tests/conftest.py

```diff
@@ -73,16 +82,48 @@ def client_ctx(app):  # pylint: disable=redefined-outer-name
         yield _client
 
 
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="session")
 def db(app):  # pylint: disable=redefined-outer-name, invalid-name
-    """Return a session-wide initialised database."""
+    """Return a session-wide initialised database.
```

Comment:
Why is this being changed?

Reply:
was done the other way in mailer. will check again

Reply:
unless there's a specific reason to do this, please revert

Reply:
i added postgres to docker compose, it looks like running it that way does not like 'drop database' operation -> E           pg8000.exceptions.DatabaseError: {'S': 'ERROR', 'V': 'ERROR', 'C': '55006', 'M': 'cannot drop the currently open database', 'F': 'dbcommands.c', 'L': '1586', 'R': 'dropdb'} ---- hence the switch to drop schema from (which was used in the mailer). I found having db in the docker compose convenient for running unit tests, potentially i can revert it.

Reply:
it's a common mistake, don't use a postgres named database